graphs/util/db.py

```diff
@@ -1608,6 +1608,300 @@ def find_all_graphs_containing_nodes(uid, search_type, search_word, view_type, d
 
 	return graph_dict.values()
 
+
+def uploadGPMLFile(username, graphJSON, title):
```

Comment:
Thanks. I will review it today.
On Jun 16, 2016 9:55 AM, "Mridul Seth" <notifications@github.com> wrote:

> In graphs/util/db.py
> <https://github.com/Murali-group/GraphSpace/pull/162#discussion_r67348160>
> :
>
> > @@ -1608,6 +1608,300 @@ def find_all_graphs_containing_nodes(uid, search_type, search_word, view_type, d
> >
> >  	return graph_dict.values()
> >
> > +
> > +def uploadGPMLFile(username, graphJSON, title):
>
> @adbharadwaj <https://github.com/adbharadwaj> This is where all starts :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Murali-group/GraphSpace/pull/162/files/6abc14b7009b1111f425a048143a762fcb995b77#r67348160>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADUeB5UkkZUUGXpIM8ly3K-1ff2kLdTtks5qMVXigaJpZM4I3aAF>
> .
>